tool/tsh/common/tsh_test.go

```diff
@@ -3524,6 +3524,7 @@ func TestSerializeDatabases(t *testing.T) {
         "rds": {
           "iam_auth": false
         },
+		"iam_policy_exists": false,
```

Comment:
This new entry seems weirdly indented 😓

Reply:
Thank you, I'll fix it.
My editor failed me, I'm disappointed.